### PR TITLE
re-enable multithreaded mixed 3D assembly

### PR DESCRIPTION
**Pull request by akva2**
the problems leading to it being disabled should have been fixed now

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this with downstreams please

File: src/ASM/ASMs3Dmx.C [hunk 1]
```diff
@@ -1251,9 +1251,6 @@ void ASMs3Dmx::generateThreadGroups (const Integrand& integrand, bool silence,
 
 void ASMs3Dmx::generateThreadGroups (char lIndex, bool silence, bool)
 {
-#ifdef USE_OPENMP
-  omp_set_num_threads(1);
-#endif
```
Comment (kmokstad) [inline src/ASM/ASMs3Dmx.C:1256]: But then you could just remove the overriding and just use the parent class method, since it be just the same anyway?

Comment (akva2) [reply src/ASM/ASMs3Dmx.C:1256]: yeah, right you are.

**Comment by akva2** [comment]
jenkins build this with downstreams please

**Pull request merged**